stbt-docker

```diff
@@ -382,6 +382,7 @@ def main(argv):
               -o -print0 | xargs -0 chown -f {uid}:{gid} &&
          sed -i s/stb-tester:x:1000:1000/stb-tester:x:{uid}:{gid}/ /etc/passwd &&
          sed -i s/stb-tester:x:1000/stb-tester:x:{gid}/ /etc/group &&
+         ln -s /dev/null /dev/raw1394 &&
```

Comment:
This does make me a little nervous.  I'm thinking about the case where you have `/dev` mounted in from the host.  Maybe it's fine.  Also I don't think we would want `stbt-docker` to fail if `/dev/raw1394` existed or `/dev` was for some reason unwritable.